script.js

```diff
@@ -1,19 +1,252 @@
+    h1.appendChild(h1_t);
+    section2.appendChild(h1);
+    section.appendChild(section2);
+    append_child_center(section);
+}
+
+function create_content(section2, section, p) {
+    let p_t = create_text("Lorem ipsum dolor sit amet, consectetur adipisicing elit. Commodi voluptates, nulla porro accusamus aperiam incidunt delectus quisquam totam unde, velit dolore minima. At doloremque ea eius iste fugiat dolorem sapiente? Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos ducimus perspiciatis nesciunt vero mollitia voluptatum beatae, neque molestias, doloremque harum at impedit nobis optio! Quisquam corporis officia fugiat aliquid tenetur?");
+    p.setAttribute("id", "text");
+    p.appendChild(p_t);
+    section2.appendChild(p);
+    section.appendChild(section2);
+    append_child_center(section);
+}
+
+function create_button(section2, section, paper_name) {
+    let button = create_element("button", "button", paper_name);
+    let button_t = create_text("Continue Reading");
+    button.appendChild(button_t);
+    section2.appendChild(button);
+    section.appendChild(section2);
+    append_child_center(section);
+    button.setAttribute("onclick", "modal_window(id)");
+}
+
+function create_border(section) {
+    let border = create_element("div", "border", "border");
+    section.appendChild(border);
+    append_child_center(section);
+}
+
+function create_dropdown(name) {
+    let selectedOption = 0;
+    let h4 = create_element("h4", "", "");
+    let h4_t = create_text("SELECT CATEGORY");
+    h4.appendChild(h4_t);
+    append_child_side(h4);
+    let drop = create_element("select", "drop", "select");
+    let option;
+    for (let i = 0; i <= 10; i++) {
+        option = create_element("option", "", "");
+        option.setAttribute("value", arr[i]);
+        drop.setAttribute("onchange", "create_single_tile(value)");
+        let option_t = create_text(arr[i]);
+        option.appendChild(option_t);
+        drop.appendChild(option);
+        append_child_side(drop);
+        if (arr[i] == name) {
+            selectedOption = i;
+        }
+    }
+    document.getElementById("select").selectedIndex = selectedOption;
+}
+
+function create_email() {
+    h4 = create_element("h4", "", "");
+    let h4_t = create_text("SUBSCRIBE");
+    h4.appendChild(h4_t);
+    append_child_side(h4);
+    let input = create_element("input", "email", "email");;
+    input.setAttribute("type", "text");
+    input.setAttribute("placeholder", "  Email Address");
+    append_child_side(input);
+    input = create_element("input", "subscribe", "subscribe");
+    input.setAttribute("type", "submit");
+    input.setAttribute("value", "Subscribe");
+    input.addEventListener("click", function () {
+        validate_Email(document.getElementById("email").value);
+    })
+    append_child_side(input);
+}
+
+function create_single_tile(value) {
+    center.innerHTML = '';
+    if (value == "All") {
+        body_section();
```

Comment:
Here on calling this body_section will again recreates the dropdown and email elements, this should be avoided, 
better modify the body_section() with few more smaller functions and use them.